_content/monographs/logic-epistemology-theravada_hegoda-khemananda.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "Logic and Epistemology in Theravada"
+authors:
+  - "Hegoda Khemananda"
+external_url: "https://archive.org/download/logic-and-epistemology-in-theravada/Logic%20and%20Epistemology%20in%20Theravada_text.pdf"
+drive_links:
+  - "https://drive.google.com/file/d/1-7xj6Sl-G3Gw1OaQupet_yAaeX1_K1SI/view?usp=drivesdk"
+translator: "Asangha Tilakaratne"
+course: abhidhamma
+tags:
```

Comment:
There's also a lot in there about how the Buddha taught... 🤔 I wonder if either a hermeneutics or pedagogy tag might be appropriate? What do you think?

Reply:
I like it. I would go with pedagogy over hermeneutics here, simply because hermeneutics feels like it deals with commentary.